src/composite/MegaChart.css

```diff
@@ -1,3 +1,5 @@
+.composite_MegaChart-Info,
+.composite_MegaChart-Info *,
```

Comment:
Yes, I'm using css to default the font family to "FontAwesome." The same way the maximize button font family defaults.